utils.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+func HandleTransaction(method, hash string) {
```

Comment:
I don't understand your function here. it does the same if we provide `method = "POST"` or `method = "DELETE"`, while we would expect different behaviors.

Reply:
`http.NewRequest()` takes `method` as first parameter instead of `"POST"`, I fixed it.